tools/system_libs.py

```diff
@@ -1458,6 +1458,18 @@ def can_build(self):
     return super(libstandalonewasm, self).can_build() and shared.Settings.WASM_BACKEND
 
 
+class libjsmath(Library):
+  name = 'libjsmath'
+
+  cflags = ['-Os']
+  src_dir = ['system', 'lib']
+
+  def get_files(self):
+    return files_in_path(
+        path_components=['system', 'lib'],
+        filenames=['js_math.cpp'])
```

Comment:
It looks like you can just go `src_files = ['jsmath.cpp']` instead of this method.

BTW maybe call it `jsmath.cpp` to match the library name?